backend/package.json

```diff
@@ -11,7 +11,7 @@
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "nest start",
     "start:dev": "nest start --watch",
-    "start:debug": "nest start --debug --watch",
+    "start:debug": "nest start --debug=0.0.0.0:9999 --watch",
```

Comment:
Opened up a remote debug port on the NestJS backend running out of the docker container locally.